File: 1470. Shuffle the Array/solution.java
class Solution {
  public int[] shuffle(int[] nums, int n) {
    int[] result = new int[nums.length];
    for (int index = 0; index < n; index++){
      result[index * 2] = nums[index];
      result[(index * 2) + 1] = nums[n + index];
    }
    
    return result;
  }
}

File: 1470. Shuffle the Array/solution.py
class Solution(object):
  def shuffle(self, nums, n):
    """
    :type nums: List[int]
    :type n: int
    :rtype: List[int]
    """
    result = []
    for index in range(n):
      result.append(nums[index])
      result.append(nums[n + index])
    
    return result